.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+*       @bbc/simorgh
```

Comment:
The original intent of the Dependabot PR was to add the Simorgh team as a review on those PRs only. This is because we have removed the noisy posts from our code review channel, so didn't want them to go unnoticed. 

Given that Dependabot PRs tend to update package.json files and the yarn cache, I wonder if we only add a codeowner to these files/directories? 

On the other hand, I don't see any harm in adding a blanket ownership across the repo, and for all members being notified for all PRs? This will also notify us of PRs opened by external contributors.

Reply:
oh interesting ! yeah i'd quite like to discuss it at next tech session - would be good to go over dependency review process now i see you've done some cool updates to how they come through in the review channel

Reply:
> i see you've done some cool updates to how they come through in the review channel

Hahaa thanks! Not quite finished but was absolutely planning to share wider once I had finished messing about with it 😸